src/resources/help/en_US/relnotes3.4.0.html

```diff
@@ -72,7 +72,8 @@
 <a name="Query_changes"></a>
 <p><b><font size="4">Changes to VisIt's picks and queries in version 3.4</font></b></p>
 <ul>
-  <li>feature1</li>
+  <li>The X Ray Image Query no longer has a family output files option. Instead, this has been replaced with a choice of three "filenaming schemes". The default is "none", which will attempt to keep the filename as simple as possible. There is an option to "family" output files, which has nearly the same behavior as before. The final option is to have "cycle" information in filenames.</li>
```

Comment:
First sentance conflicts with second. There still is that option. It is just implemented differently.

```suggestion
  <li>The X Ray Image Query has been modified to support 3 different file naming schemes. The default is "none", which will attempt to keep the filename as simple as possible. There is an option to "family" output files, which has nearly the same behavior as before. The final option is to have "cycle" information in filenames.</li>
```

Reply:
Oops, you are right! Initially I removed the `family_files` option and did the release notes and everything, but I had a change of heart and forgot to update this.